zuul-core/src/main/java/com/netflix/zuul/netty/server/http2/Http2OrHttpHandler.java

```diff
@@ -70,6 +75,42 @@ public Http2OrHttpHandler(
         this.addHttpHandlerFn = addHttpHandlerFn;
     }
 
+    @Override
+    public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
```

Comment:
nit: add a javadoc saying this method is inspired by `ApplicationProtocolNegotiationHandler.userEventTriggered`